### ISSUE TITLE
Any change for a release on pypi ?

### ISSUE THREAD

**Issue by TTRh** [aboSamoor/polyglot#205]
i'm interested to get some latest fixes, but latest release on pypi is pretty old ... Any chance to push a new release soon ?
Thanks !

**Comment by thatch** [comment]
Indeed, the sdist on pypi doesn't have the python3-compatible setup.py and thinks there are no deps when building a wheel.
